t/config.t

```diff
@@ -193,8 +194,6 @@ subtest 'trim whitespace characters from both ends of openqa.ini value' => sub {
     ';
     $t_dir->child('openqa.ini')->spurt($data);
     OpenQA::Setup::read_config($app);
-    print $app->config->{global}->{appname};
-    print $app->config->{global}->{hide_asset_types};
```

Comment:
unrelated changes (including the whitespace changes above). Better create a separate commit. Can still be part of this PR but separate commit.